c2corg_ui/static/js/alertservice.js

```diff
@@ -26,6 +26,20 @@ app.Alerts = function(gettextCatalog) {
 
 
 /**
+ * Use this function to annotate a string for extraction by the
+ * gettext extract tool. The call to this function will be eliminated
+ * by the Closure compiler when minifying, so it should not have any
+ * performance effect.
+ * See https://angular-gettext.rocketeer.be/dev-guide/annotate-js/
+ * @param {string} str String to have extracted by gettext tool
+ * @return {string}
+ */
+app.Alerts.prototype.gettext = function(str) {
```

Comment:
Is this function supposed to be used only for alerts? If it is a generic function, maybe it would make sense to put it into some utils module and not on `app.Alerts`.